src/general/MainMenu.tscn

```diff
@@ -898,7 +898,6 @@ visible = false
 visible = false
 
 [node name="NewGameSettings" parent="." instance=ExtResource( 18 )]
-visible = false
```

Comment:
Unintended change to this being visible by default.

Reply:
This line of code is still here...

Reply:
Just commenting again to add this (and my previous comment) to my current review.